modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -383,12 +383,13 @@ The API specification downloaded from API Community Manager is the same as the s
 
 The available buttons are *Download Spec as RAML* and *Download Spec as OAS*.
 
-Specifications may be uploaded as either RAML or OAS, and the other version is automatically generated. Due to small differences in the RAML and OAS formats, the automatically generated version might have small inconsistencies with the uploaded version.
+Specifications may be uploaded as RAML, OAS 2.0, or OAS 3.0. When a RAML version is uploaded, an OAS 2.0 version is automatically generated. When an OAS 2.0 or OAS 3.0 version is uploaded, a RAML version is automatically generated. Due to small differences in the RAML and OAS formats, the automatically generated version might have small inconsistencies with the uploaded version.
 
 You can specify values for the following parameters of the Spec Downloader component:
 
 Record Id:: Specifies the API. Default is `{!recordId}`, which downloads the specification of the API selected on the current page.
 Layout:: Specifies the layout of the buttons, *Horizontal* (default) or *Vertical*.
+Buttons Color Picker:: Specifies the color of the buttons.
```

Comment:
If these are in the UI as labels, they should probably be formatted * *

Reply:
Good point. I'll change this throughout the file in a future PR.